### PR TITLE
feat: add e2e tests

### LINKED ISSUE #85: Add Cypress and first E2E tests

### PR DESCRIPTION
**Pull request by pedronauck**
✨ Fix #85

### CODE REVIEW

File: .github/actions/setup-rust/action.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: 'Setup Rust env'
```
Comment (luizstacio) [inline .github/actions/setup-rust/action.yaml:1]: I think we can remove this part as we don't build contracts on the wallet project

File: packages/app/cypress/support/e2e.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+// 'supportFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
+import './commands';
+
+// Alternatively you can use CommonJS syntax:
```
Comment (luizstacio) [inline packages/app/cypress/support/e2e.ts:19]: I think we can remove as we just need ES2015

**Review by luizstacio — CHANGES_REQUESTED** [review]

File: packages/app/cypress/support/e2e.ts (hunk 2 above)
Comment (pedronauck) [reply packages/app/cypress/support/e2e.ts]: ✅

File: .github/actions/setup-rust/action.yaml (hunk 1 above)
Comment (pedronauck) [reply .github/actions/setup-rust/action.yaml]: ✅

**Review by luizstacio — APPROVED** [review]

**Pull request merged**